scripts/qa_crowbarsetup.sh

```diff
@@ -1897,10 +1897,8 @@ function enable_ssl_generic
             $p "$a['api']['protocol']" "'https'"
         ;;
         ceph)
-            $p "$a['radosgw']['ssl']['enabled']" true
-            $p "$a['radosgw']['ssl']['generate_certs']" true
-            $p "$a['radosgw']['ssl']['insecure']" true
-            return
+            a+="['radosgw']"
```

Comment:
I see the optimisation (-2 lines), but it turns the generic changes (below) into special cases. This is very hidden.
$somewhen this will hit us.